src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java

```diff
@@ -67,13 +67,42 @@
  * &lt;module name="JavadocVariable"/&gt;
  * </pre>
  * <p>
+ * By default, this setting will report a violation if
+ * there is no javadoc for any scope member.
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private int a; // violation, missing javadoc for private member
+ *
+ *   &#47;**
+ *   * Some description here
+ *   *&#47;
```

Comment:
make it to be rendered as:
```
/**
 * Some description here
 */
```

leading space for 2nd and 3rd line

Reply:
done